### PR TITLE
Fix gc collection issue

### PR DESCRIPTION
**Pull request by mary-dcouto**
keep delegates alive until ziti service is stopped

### CODE REVIEW

File: Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs [hunk 1]
```diff
@@ -150,17 +150,18 @@ public class ZitiIdentity {
...
 				throw new System.InvalidOperationException("The identity is already running");
 			}
 			
 			new Thread(() => Native.API.z4d_uv_run(Loop.nativeUvLoop)).Start();
 			await runlock.WaitAsync().ConfigureAwait(false);
+			GC.KeepAlive(ziti_opts);
 		}
 
-		public void Configure(int refreshInterval) {
+		public Native.ziti_options Configure(int refreshInterval) {
```
Comment (RodneyRichardson) [inline Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs]: I have a question on style: I personally wouldn't put any "Native" types into the public interface of the C# classes. I would suggest either making Configure private, or creating a ZitiOptions wrapper around Native.ziti_options.

Comment (dovholuknf) [reply Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs]: @RodneyRichardson - 100% agree. The public API needs to be 100% idiomatic C# to keep all that sort of internals hidden. I've not been able to get through @mary-dcouto 's PRs myself lately but that's a great catch!

Comment (mary-dcouto) [reply Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs]: Thanks for the review/comments. I changed the return type to ZitiOptions Wrapper.

Comment (dovholuknf) [reply Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs]: we'll address this in the big pr back to main. this still returns a 'native' struct from managed code... we'll fix this on a different branch. right now i'm condensing all these PRs to one big one

**Pull request merged**